partitura/io/mei_export_v2.py

```diff
@@ -0,0 +1,383 @@
+                note_el = measure_el.xpath(f".//*[@xml:id='{note.id}']")
+                if len(note_el) > 0:
+                    note_el = note_el[0]
+                    layer_el = note_el.getparent()
+                    insert_index = layer_el.index(note_el)
+                    # Create the clef element
+                    clef_el = etree.Element('clef')
+                    layer_el.insert(insert_index, clef_el)
+                    clef_el.set(XMLNS_ID, "clef-" + self.elc_id())
+                    clef_el.set('shape', str(clef.sign))
+                    clef_el.set('line', str(clef.line))
+
+    def _handle_ks_changes(self, measure_el, start, end):
+        # For key signature changes, we add a new scoreDef element at the beginning of the measure
+        # and add the key signature element as attributes of the scoreDef element
+        for key_sig in self.part.iter_all(spt.KeySignature, start=start, end=end):
+            if key_sig.start.t == 0:
+                continue
+            # Create the scoreDef element
+            score_def_el = etree.Element('scoreDef')
+            score_def_el.set(XMLNS_ID, "scoredef-" + self.elc_id())
+            score_def_el.set('mode', key_sig.mode) if key_sig.mode is not None else score_def_el.set('mode', 'major')
+            if key_sig.fifths == 0:
+                score_def_el.set('sig', '0')
+            elif key_sig.fifths > 0:
+                score_def_el.set('sig', str(key_sig.fifths) + 's')
+            else:
+                score_def_el.set('sig', str(abs(key_sig.fifths)) + 'f')
+            # Find the pname from the number of sharps or flats and the mode
+            score_def_el.set('pname', fifths_mode_to_key_name(key_sig.fifths, key_sig.mode).lower())
+            # Add the scoreDef element at before the measure element starts
+            parent = measure_el.getparent()
+            parent.insert(parent.index(measure_el), score_def_el)
+
+    def _handle_ts_changes(self, measure_el, start, end):
+        # For key signature changes, we add a new scoreDef element at the beginning of the measure
+        # and add the key signature element as attributes of the scoreDef element
+        for time_sig in self.part.iter_all(spt.TimeSignature, start=start, end=end):
+            if time_sig.start.t == 0:
+                continue
+            # Create the scoreDef element
+            score_def_el = etree.Element('scoreDef')
+            score_def_el.set(XMLNS_ID, "scoredef-" + self.elc_id())
+
+            # Add the scoreDef element at before the measure element starts
+            parent = measure_el.getparent()
+            parent.insert(parent.index(measure_el), score_def_el)
+            score_def_el.set('count', str(time_sig.beats))
+            score_def_el.set('unit', str(time_sig.beat_type))
+
+    def _handle_harmony(self, measure_el, start, end):
+        # For key signature changes, we add a new scoreDef element at the beginning of the measure
```

Comment:
is this comment for another function?

Reply:
Adressed with commit [273fd32](https://github.com/CPJKU/partitura/pull/347/commits/273fd325fcbc97a9a6fbbd3b74792e5efb3f6d42)